src/auro-dropdown.js

```diff
@@ -94,7 +94,32 @@ class AuroDropdown extends LitElement {
     this.dropdownWidth = this.getBoundingClientRect().width;
   }
 
+  /**
+   * @private
+   * @returns {void} Updates the trigger `aria-label` or `aria-labeledby`.
+   */
+  processAriaLabel() {
+    this.trigger = this.shadowRoot.querySelector(`#trigger`);
+
+    if (this.getAttribute('aria-label')) {
+      // If custom aria-label declared at root apply it to the trigger
+      this.trigger.setAttribute('aria-label', this.getAttribute('aria-label'));
+    } else if (this.getAttribute('aria-labeledby')) {
+      // Else if custom aria-labeledby declared at root apply it to trigger
+      this.trigger.setAttribute('aria-labeledby', this.getAttribute('aria-labeledby'));
```

Comment:
I'm not sure if this will be valid. We should test this behavior with a screen reader. My concern is that `<auro-dropdown aria-labelledby="foo">` will refer to an element with ID "foo" outside of this component's shadow root, and using `aria-labelledby="foo"` inside the shadow root won't be able to reference that element.

Reply:
This works correctly for me using VoiceOver on my mac. If you run the localhost demo page that last example in the demo renders with this method. @geoffrich please let me know if your mileage is different with this in the windows environment! I'd appreciate the feedback after trying out that coded example.

Reply:
This only worked once I added `role="button"` to the trigger. Otherwise, NVDA wouldn't announce the labels (presumably because the trigger wasn't exposed as an interactive element). Can you make that change?